docs/products/kafka/best-practices.rst

```diff
@@ -0,0 +1,75 @@
+Best practices
+==============
+
+We recommend to follow these best practices to ensure that your Apache Kafka service is fast and reliable.
+
+Apache Kafka
+-------------
+
+Check your topic replication factors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Apache Kafka services rely on replication between brokers to preserve data in
+case of the loss of a node. Consider how business critical the data in
+each topic is and make sure that replication is set high enough for it.
+
+You can set the replication factor in `Aiven web console <https://console.aiven.io/>`_ when you create a new topic or edit an existing one.
+
+.. note:: Do not set the replication factor below 2 in order to prevent data loss from unexpected node termination.
```

Comment:
Do we allow replication factor to be less than 1? IIRC we don't

Reply:
We show error in this case. But I think it still nice to explain here why.
"ERROR
Replication factor must be set to at least two"

Reply:
Changed the wording